js/components/ImportIndicatorsPopover.js

```diff
@@ -0,0 +1,122 @@
+        console.log("Upload Clicked");
+        api.uploadTemplate()
+        .then(response => {
+            console.log("Reponse:", response);
+            alert("UPLOAD TEMPLATE")
+        })
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                {(() => {
+                    switch(this.state.status) {
+                        case this.ONE:
+                            return (
+                                <div className="importIndicators-body">
+                                    <div className="import-text">
+                                        <ol>
+                                            <li>
+                                                {
+                                                    // # Translators: TODO
+                                                    gettext("Download the template, open it in Excel, and enter indicators.")
+                                                }
+                                            </li>
+                                            <li>
+                                                {
+                                                    // # Translators: TODO
+                                                    gettext("Upload the template and follow instructions to complete the proocess.")
+                                                }
+                                            </li>
+                                        </ol>
+                                    </div>
+
+                                    <div className="import-buttons">
+                                        <button
+                                            type="button"
+                                            className="btn btn-sm btn-primary btn-download"
+                                            onClick={() => this.handleDownload()}
+                                            >
+                                            {
+                                                // # Translators: TODO
+                                                gettext("Download template")
+                                            }
+                                        </button>
+                                        <button
+                                            type="button"
+                                            className="btn btn-sm btn-primary btn-upload"
+                                            onClick={() => this.handleUpload()}
+                                        >
+                                            {
+                                                // # Translators: TODO
+                                                gettext("Upload template")
+                                            }
+                                        </button>
+                                    </div>
+                                </div> 
+                            );
+                        case this.TWO:
```

Comment:
Is this a placeholder for something?

Reply:
Yes, this was a placeholder for the other views of the popover. I updated this in the latest commit to give a descriptive name for these views: Download_Upload, Feedback, Confirm, and Success views.